snapcraft.yaml

```diff
@@ -115,6 +119,8 @@ parts:
       - libgl1-mesa-glx
       - libsdl2-2.0-0
       - libsdl2-gfx-1.0-0
+    install: |
+      make test
```

Comment:
The funny thing is, we don't override, its running after the actual installation step.